docs/pages/enterprise/sso.mdx

```diff
@@ -129,8 +129,11 @@ or </ScopedBlock>create a `cluster_auth_preference` resource.
 </Tabs>
 
 
-Next, define an authentication connector. Create a file called `connector.yaml`
-based on one of the following examples.
+Next, define an authentication connector. We recommend the usage of helper 
```

Comment:
What would you think of adding step-by-step instructions for using a helper command to create the connector? We're already showing example snippets, so I think it would make sense to create a seamless step-by-step experience. We could use a `Tabs` component for this, with one `TabItem` per connector type.

Reply:
Hmm, not sure what that should look like? Do you mean to move individual steps in this guide into tabs? I'm probably missing the right idea here...

Reply:
It would be something like this:

````markdown
Next, define an authentication connector using the `tctl sso configure` 
command:

<Tabs>
  <TabItem title="OIDC">

  Here is how to use `tctl sso configure oidc`.

  </TabItem>
  <TabItem title="SAML">

  Here is how to use `tctl sso configure saml`.

  </TabItem>
  <TabItem title="GitHub">

  Here is how to use `tctl sso configure gh`.

  </TabItem
</Tabs>
````
This way, the reader wouldn't need to find the relevant documentation on their own.